### PR TITLE
Blog: Introduce Lucky 0.15 and new website

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: src/posts/introducing_the_lucky_blog.cr [hunk 1]
```diff
@@ -1,129 +1,33 @@
...
-      ```crystal
-        add username : String, unique: true
-      ```
-
-    * `create_index`: Adds an index to a specified column.
-
-      ```crystal
-      create_index :users, :last_name, unique: true
-      ```
+    ## New class based components
```
Comment (paulcsmith) [inline src/posts/introducing_the_lucky_blog.cr]: So far I want to highlight new class based components and the new security featueres. What else would be good to highlight? Asset URL prefix for CDN?

@jwoertink @edwardloveall

File: src/posts/introducing_lucky_0.15.cr [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    goodies, bug fixes, and support for the newest version of Crystal (0.29.0)
+
+    We're also releasing a brand new website along with this new version of Lucky.
+    The website is mobile friendly, has a blog (this one!) and is written using Lucky.
+
+    Enjoy!
+
+    ## New class based components
+
+    ## New built-in security features
```
Comment (paulcsmith) [inline src/posts/introducing_lucky_0.15.cr]: I decided to rename the file so the diff was easier to read. Here is where I plan to highlight new stuff

Thoughts on what should be included?

File: src/posts/introducing_the_lucky_blog.cr (hunk 1 above)
Comment (paulcsmith) [reply src/posts/introducing_the_lucky_blog.cr]: Moved convo here: https://github.com/paulcsmith/website-v2/pull/55#discussion_r292246980

File: src/posts/introducing_lucky_0.15.cr (hunk 2 above)
Comment (jwoertink) [reply src/posts/introducing_lucky_0.15.cr]: Maybe you can mention it similar to how rails does it like:

### Lucky framework
fallback routing
security header modules
some bug fixes

### Avram
sequences

### Router
### Flow
### Documentation
a new websites...

**Comment by paulcsmith** [comment]
I'm gonna go ahead and merge this, but would love some feedback (or PRs!) later.

**Pull request merged**